radio/src/gui/colorlcd/menu_model.cpp

```diff
@@ -74,3 +75,13 @@ void ModelMenu::onEvent(event_t event)
   }
 #endif
 }
+
+void ModelMenu::addButton(TabsGroupHeader* header)
+{
+  OpenTxTheme::instance()->createTextButton(
+      header, {LCD_W / 2 + 6, MENU_TITLE_TOP + 1, LCD_W / 2 - 8, MENU_TITLE_HEIGHT - 2},
+      STR_OPEN_CHANNEL_MONITORS, [=]() {
+        pushEvent(EVT_KEY_FIRST(KEY_MODEL));
+        return 0;
+      });
+}
```

Comment:
I think the name of that function is very generic for such a specific thing it does.
Maybe add something to the tab carousel to add a button there, with a lambda to trigger an action?

Reply:
I have changed the name of the function, but I do not think it makes sense to do it with lambda: This functionality is completely internal to ModelMenu and nobody outside it should know about its existence. ModelMenu is constructed in two different places and the same lambda-expression will have to be written twice.
So, especially taking into account that there is no need to differentiate targets with and without HW buttons, I suggest to keep current implementation.

Reply:
I can live with the changed name